lib/elixir/lib/calendar/iso.ex

```diff
@@ -464,6 +464,10 @@ defmodule Calendar.ISO do
   @doc """
   Converts the given time into a string.
 
+  By default, returns times formatted in the "extended" format,
+  for human readability. It also supports the "basic" format
+  through passing the `:basic` option.
```

Comment:
```suggestion
  by passing the `:basic` option.
```

Reply:
This was copied from here:

https://github.com/elixir-lang/elixir/blob/3bd3f88d57d7fff6cab7b171294b89fb08eedfe7/lib/elixir/lib/calendar/datetime.ex#L651-L652

We should probably change those in a different PR